modules/bootcamp/detect_landing_pad.py

```
@@ -86,33 +86,36 @@ def run(self, image: np.ndarray) -> "tuple[list[bounding_box.BoundingBox], np.nd
         # * conf
         # * device
         # * verbose
-        predictions = ...
+        predictions = self.__model.predict(image, conf=0.7, device=self.__DEVICE, verbose=False)
 
         # Get the Result object
-        prediction = ...
+        prediction = predictions[0] # Get the (only) result at index 0
 
         # Plot the annotated image from the Result object
         # Include the confidence value
-        image_annotated = ...
+        # We take the result and turn it into a numpy array image
+        image_annotated = prediction.plot() 
 
         # Get the xyxy boxes list from the Boxes object in the Result object
-        boxes_xyxy = ...
+        # Take the Result object and turn it into a list of coordinate lists populated with xy coords
+        boxes_xyxy = prediction.boxes.xyxy
 
         # Detach the xyxy boxes to make a copy,
         # move the copy into CPU space,
         # and convert to a numpy array
-        boxes_cpu = ...
+        boxes_cpu = boxes_xyxy.cpu().numpy()
 
         # Loop over the boxes list and create a list of bounding boxes
         bounding_boxes = []
-        # Hint: .shape gets the dimensions of the numpy array
-        # for i in range(0, ...):
-            # Create BoundingBox object and append to list
-            # result, box = ...
 
-        # Remove this when done
-        raise NotImplementedError
+        for xyxy in boxes_cpu:
+            result, box = bounding_box.BoundingBox.create(xyxy)
+            if result:
+                bounding_boxes.append(box)
```

instead of just skipping cases where box creation fails, we actually want to return nothing if any box creation fails
this is because in data science we want to skip the entirety of a corrupted data point than to return part of it (the image is the data point, the boxes are the "parts")